test/new-e2e/tests/containers/docker_test.go

```diff
@@ -149,6 +153,44 @@ func (suite *DockerSuite) TestDockerMetrics() {
 	})
 }
 
+func (suite *DockerSuite) TestDockerEvents() {
+	const ctrNameSize = 12
+	const ctrNameCharset = "abcdefghijklmnopqrstuvwxyz0123456789"
+
+	rand.Seed(time.Now().UnixNano())
+
+	ctrNameData := make([]byte, ctrNameSize)
+	for i := range ctrNameSize {
+		ctrNameData[i] = ctrNameCharset[rand.Intn(len(ctrNameCharset))]
+	}
+	ctrName := "exit_42_" + string(ctrNameData)
+
+	suite.Env().RemoteHost.MustExecute(fmt.Sprintf("docker run -d --name \"%s\" busybox sh -c \"exit 42\"", ctrName))
```

Comment:
Should a different `busybox` image be used to avoid rate limit errors (`public.ecr.aws/docker/library/busybox`)?

Reply:
Very good point!
I fixed that in https://github.com/DataDog/datadog-agent/pull/34210/commits/119d64799ccb10f7a8db8f446ad3f93457982bc2.